voyager/modules/client/ethereum/src/main.rs

```diff
@@ -47,7 +47,7 @@ impl ClientModule for Module {
     async fn new(config: Self::Config, info: ClientModuleInfo) -> Result<Self, BoxDynError> {
         info.ensure_client_type(ClientType::ETHEREUM)?;
         info.ensure_consensus_type(ConsensusType::ETHEREUM)?;
-        info.ensure_ibc_interface(IbcInterface::IBC_COSMWASM)?;
+        info.ensure_ibc_interface(vec![IbcInterface::IBC_COSMWASM])?;
```

Comment:
if you change this to accept `impl IntoIterator<Item = impl AsRef<str>>`, you can simply pass `[IbcInterface::IBC_COSMWASM]` here